save-cloud-common/src/commonMain/kotlin/org/cqfn/save/entities/Project.kt

```diff
@@ -5,29 +5,38 @@ import org.cqfn.save.utils.EnumType
 import kotlinx.serialization.Serializable
 
 /**
- * @property owner
  * @property name
  * @property url
  * @property description description of the project, may be absent
  * @property status status of project
  * @property public
  * @property userId the user that has created this project. No automatic mapping, because Hibernate is not available in common code.
- * @property adminIds comma-separated list of IDs of users that are admins of this project
- * @property organizationId
+ * @property organization
  */
 @Entity
 @Serializable
 data class Project(
-    var owner: String,
     var name: String,
     var url: String?,
     var description: String?,
     @Enumerated(EnumType.STRING)
     var status: ProjectStatus,
     var public: Boolean = true,
     var userId: Long? = null,
-    var adminIds: String? = null,
-    var organizationId: Long? = null,
+
+    @ManyToOne
```

Comment:
Again, we can keep `Project` a data class as long as we use eager associations. But it's fine until #363

Reply:
Ok. Let's refactor it after this